src/wallet/wallet.cpp

```diff
@@ -4311,6 +4311,49 @@ bool CWallet::BackupWallet(const std::string& strDest)
     return database->Backup(strDest);
 }
 
+bool CWallet::PendingNameFirstUpdateExists(const std::string &name)
+{
+    LOCK(cs_wallet);
+    return namePendingMap.end() != namePendingMap.find(name);
```

Comment:
An alternative would be `namePendingMap.count(name) > 0`.  I don't really care which one, though, so feel free to keep the current version.  (I just wanted to let you know about `count` in case you didn't know it yet.)

Reply:
Fixed, thanks for the tip.